server/game/cards/06-CotE/StudentOfWar.js

```diff
@@ -0,0 +1,17 @@
+const DrawCard = require('../../drawcard.js');
+
+class StudentOfWar extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
```

Comment:
this should just use the new `this.composure()` method

Reply:
Ah, didn't realise that was a thing.  I have submitted a new commit.